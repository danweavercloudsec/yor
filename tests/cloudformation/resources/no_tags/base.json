{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "yor_trace",
            "Value": "09348f8d-34ba-4392-a459-d38245588f4c"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}