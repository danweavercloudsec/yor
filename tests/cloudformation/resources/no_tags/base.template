AWSTemplateFormatVersion: 2010-09-09
Resources:
  S3Bucket:
    Type: AWS::S3::Bucket
    Properties:
      AccessControl: PublicRead
      WebsiteConfiguration:
        IndexDocument: index.html
        ErrorDocument: error.html
      Tags:
        - Key: yor_trace
          Value: e892cc08-054f-4960-9f9b-7ada7495af5d
    DeletionPolicy: Retain