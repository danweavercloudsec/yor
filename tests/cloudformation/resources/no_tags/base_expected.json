{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "new_tag",
            "Value": "new_value"
          },
          {
            "Key": "another_tag",
            "Value": "another_val"
          },
          {
            "Key": "yor_trace",
            "Value": "95e4b8c0-ba12-4bb4-a288-2220176ddabf"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}